openrouteservice/src/main/java/heigit/ors/api/requests/matrix/MatrixRequestHandler.java

```diff
@@ -97,11 +100,12 @@ public static int convertMetrics(MatrixRequestEnums.Metrics[] metrics) throws Pa
         return combined;
     }
 
-    protected static Coordinate[] convertLocations(List<List<Double>> locations) throws ParameterValueException {
+    protected static Coordinate[] convertLocations(List<List<Double>> locations, int numberOfRoutes) throws ParameterValueException, ServerLimitExceededException {
         if (locations == null || locations.size() < 2)
             throw new ParameterValueException(MatrixErrorCodes.INVALID_PARAMETER_VALUE, MatrixRequest.PARAM_LOCATIONS);
-        if (locations.size() > MatrixServiceSettings.getMaximumLocations(false))
-            throw new ParameterValueException(MatrixErrorCodes.PARAMETER_VALUE_EXCEEDS_MAXIMUM, MatrixRequest.PARAM_LOCATIONS);
+        int maxLocationsCombo = MatrixServiceSettings.getMaximumRoutes(false);
```

Comment:
you don't need to declare the maxLocationsCombo variable
```java
if (numberOfRoutes > MatrixServiceSettings.getMaximumRoutes(false)) {
...
}
```

Reply:
yeah, a relict from a different error I used before